src/main/java/spoon/support/compiler/jdt/JDTTreeBuilder.java

```diff
@@ -688,6 +688,26 @@ public CtPackageReference getPackageReference(PackageBinding reference) {
 
 		final Map<TypeBinding, CtTypeReference> bindingCache = new HashMap<>();
 
+		public <T> CtTypeReference<T> getTypeReference(TypeReference[] references, TypeReference ref) {

```

Comment:
> Can you add JavaDoc or rename the parameter?
No really, I copy the behaviour from the orignal one, but I do not know the specification of the method. 
Please, adds the JavaDoc to orignal one and I will adapt it.